### PR TITLE
	Map Princeton MODS metadata

### LINKED ISSUE #161: Implement Princeton MODS mapping

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #161

### CODE REVIEW

File: spec/import/mods_transform_spec.rb [hunk 1]
```diff
@@ -4,7 +4,13 @@
 
 RSpec.describe 'Transforming MODS files' do
   let(:provide) do
-    { 'identifier' => identifier, 'exhibit_slug' => slug, 'inst_id' => 'stanford' }
+    {
```
Comment (jcoyne) [inline spec/import/mods_transform_spec.rb]: I don't think this block is used now that you've moved the definition to line 33.

File: lib/traject/macros/dlme.rb [hunk 2]
```diff
@@ -14,6 +14,24 @@ def transform_values(context, hash)
       end
     end
 
+    # try a bunch of macros and short-circuit after one returns values
+    def first(*args)
```
Comment (jcoyne) [inline lib/traject/macros/dlme.rb]: Can we call `args` something more descriptive? How about `macros`?

Comment (cbeer) [inline lib/traject/macros/dlme.rb]: 👌

File: spec/import/mods_transform_spec.rb (hunk 1 above)
Comment (cbeer) [inline spec/import/mods_transform_spec.rb]: You're right; I've switched things around a little bit now so it does get used. Keeping it around will be useful when we add additional MODS mappings.

**Pull request merged**